.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+name: build
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '7.4'
+                    tools: composer:v1
+
+            -   name: Git checkout
+                uses: actions/checkout@v2
+
+            -   name: Validate composer.json and composer.lock
+                run: composer validate
+
+            -   name: Prepare Composer cache vars
```

Comment:
You could simplify this steps by using https://github.com/ramsey/composer-install

Reply:
Done in #152